src/tab/Tabs.stories.ts

```diff
@@ -0,0 +1,445 @@
+import { within } from '@testing-library/dom';
+import userEvent from '@testing-library/user-event';
+import * as jest from 'jest-mock';
+import { html, nothing } from 'lit';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import expect from '../utils/ExpectDOM.js';
+import { assignToSlot, ComponentStoryFormat, CSFIdentifier, querySelectorAsync, raw, getSourceFromLit } from '../utils/StoryUtils.js';
+
+import { Tab } from './Tab.js';
+import { TabHeader } from './TabHeader.js';
+import { Tabs } from './Tabs.js';
+import '../label/Label.js';
+import './TabHeader.js';
+import './Tab.js';
+import './Tabs.js';
+import '../icon/Icon.js';
+
+export default {
+    title: 'UI Components/Tabs',
+    component: 'omni-tabs',
+    argTypes: {
+        header: {
+            control: 'text'
+        }
+    }
+} as CSFIdentifier;
+
+interface Args {
+    header: string;
+    '[Default Slot]': string;
+}
+
+export const Interactive: ComponentStoryFormat<Args> = {
+    render: (args: Args) => html`
+    <omni-tabs data-testid='test-tabs'>
+        ${args.header ? html`${'\r\n'}${unsafeHTML(assignToSlot('header', args.header))}` : nothing}
+        ${args['[Default Slot]'] ? html`${'\r\n'}${unsafeHTML(args['[Default Slot]'])}` : nothing}
+    </omni-tabs>
+`,
+    frameworkSources: [
+        {
+            framework: 'Vue',
+            load: (args) =>
+                getSourceFromLit(Interactive!.render!(args), undefined, (s) =>
+                    s.replace(' active', ' :active="true"').replace(' disabled', ' :disabled="true"')
+                )
+        }
+    ],
+    name: 'Interactive',
+    description: () => html`
+        <p>
+        The <code class="language-html">&lt;omni-tabs&gt;</code> component will display content based on the nested <code class="language-html">&lt;omni-tab&gt;</code> component(s). 
+        </p>
+        <p>
+        Headers are managed by either setting the <code>header</code> attribute of the <code class="language-html">&lt;omni-tab&gt;</code> component or via slotted <code class="language-html">&lt;omni-tab-header&gt;</code> component(s) that targets the <code class="language-html">&lt;omni-tabs&gt;</code> header slot by setting the <code>slot</code> attribute to <code class="language-js">"header"</code>, and displays the slotted content of the <code class="language-html">&lt;omni-tab&gt;</code> with the <code>active</code> attribute set. 
```

Comment:
This is very technical, wall of text. We need to find a way to convey what it does, instead of how it does it.

Reply:
Will remove the statement regarding the active tab as that can be conveyed by the active story. I will update the description accordingly feel free to suggest amendments.